src/middlewared/middlewared/etc_files/local/ssh/sshd_config.mako

```diff
@@ -48,6 +48,8 @@
 	users = middleware.call_sync('user.query')
 	root_user = filter_list(users, [['username', '=', 'root']], {'get': True})
 
+	login_banner = middleware.call_sync('datastore.query', 'system.advanced')[0]['adv_login_banner']
```

Comment:
You can use the endpoint directly you have in place for getting login banner only